x/superfluid/client/rest/tx.go

```diff
@@ -10,23 +10,25 @@ import (
 func ProposalSetSuperfluidAssetsRESTHandler(clientCtx client.Context) govrest.ProposalRESTHandler {
 	return govrest.ProposalRESTHandler{
 		SubRoute: "set-superfluid-assets",
-		Handler:  newSetSuperfluidAssetsHandler(clientCtx),
```

Comment:
idk whats going on here, but IIRC this rest folder doesn't matter for anything so not blocking

Reply:
For each of these, the handler is the same - no-op. I just made a shared no-op function for all of these but yes it's unimportant

Reply:
ah that makes sense!